.vsts-ci/linux.yml

```diff
@@ -80,23 +80,23 @@ jobs:
   - template: ../tools/releaseBuild/azureDevOps/templates/insert-nuget-config-azfeed.yml
 
   - powershell: |
-      tools/travis.ps1 -Stage Bootstrap
+      tools/ci.psm1 -Stage Bootstrap
```

Comment:
PSM1's don't work this way.

Look at the yml for windows.

see nothing was built or tested
https://powershell.visualstudio.com/PowerShell/_build/results?buildId=12039&view=logs

Reply:
All the other places you used a switch on the PSM need to be fixed too.

Reply:
I see, I will wrap it in a function once I have access to vscode